src/openrct2-ui/windows/TileInspector.cpp

```diff
@@ -196,6 +197,8 @@ constexpr auto GhostFlagColumnXY = ClearanceHeightColumnXY + ScreenCoordsXY{ Cle
 constexpr auto GhostFlagColumnSize = ScreenSize{ 15, 14 };
 constexpr auto LastFlagColumnXY = GhostFlagColumnXY + ScreenCoordsXY{ GhostFlagColumnSize.width, 0 };
 constexpr auto LastFlagColumnSize = ScreenSize{ 32, 14 };
+constexpr auto DirectionColumnXY = LastFlagColumnXY + ScreenCoordsXY{ LastFlagColumnSize.width, 0 };
+constexpr auto DirectionColumnSize = ScreenSize{ 32, 14 };
```

Comment:
I'd put this between the clearance height and ghost flag instead. Also reduce the width to 15, since it only shows one character in the header. The last column is wider to account for the scrollbar.